lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -55,6 +75,14 @@ module.exports = {
                 functionName
               );
             }
+            if (event.alb.authorizer) {
+              if (!this.validatedAuthorizers) {
+                throw new this.serverless.classes.Error(
+                  `authorizer passed in function "${functionName}" but no authorizers declared in provider`
+                );
+              }
+              albObj.authorizer = this.validateEventAuthorizer(event, authorizers, functionName);
```

Comment:
We're assigning an _array_ of authorizers so it should be `authorizers` and not `authorizer`